src/server/util/kubectl.cc

```diff
@@ -69,7 +90,9 @@ static std::string generate_local_object(
                     "\n  namespace: " + namespace_ +
                     "\n  labels:"
                     "\n    k8s.v6d.io/signature: " + signature +
-                    "\n    job: none"
+                    "\n    job: " + job_name +
+                    "\n    created-by-podname: " + client_pod_name +
+                    "\n    created-by-podnamespace: " + client_pod_namespace +
```

Comment:
It seems that k8s labels usually don't be named as such pattern. Using `k8s.v6d.io/created-by/name` and `k8s.v6d.io/created-by/namespace` would be better, I think.